yappi/_yappi.c

```diff
@@ -224,7 +224,7 @@ static void _DebugPrintObjects(unsigned int arg_count, ...)
 
 int 
 IS_SUSPENDED(PyFrameObject *frame) {
-#if PY_MAJOR_VERSION >= 3 && PY_MINOR_VERSION == 11
+#if PY_MAJOR_VERSION >= 3 && PY_MINOR_VERSION >= 11
```

Comment:
This would fail for a theoretical Python 4.0 (or 23.0 if it switched to CalVer).

Either check:
* major==3 and minor>=11
* or
* major>=4

Or use `PY_VERSION_HEX`:

```suggestion
// Python 3.11+
#if PY_VERSION_HEX >= 0x030B0000
```